questions.md

```diff
@@ -4,26 +4,125 @@
 
 1. On the UCI cluster, the resource request "-pe openmp 64" refers to the number of processors requested.  Does that
    request mean that your commands will use multiple processors?
+   
+   This request "-pe openmp 64" is specified to try to request all 64 cores in a parallel environment to run a program/job  on the same node.
+Yes, it will use multiple processors only if the software you are using is capable of doing so and those are available on the HPC.
+  
 2. In general, how do you know how many processors, how much RAM, how many files would be required/needed/written by the
    jobs you are running on the cluster?
+   
+The time command with the verbose option will list out this information for a command that is run
+
+```
+/usr/bin/time -v commandgoeshere
```

Comment:
this is good but it is is also good to add option -o which allows an output file. env time -v -o MYOUTPUT CMD would do the trick and do a dump to a text file. this helps sometimes if you care about what goes in your stdout and stderr files. I like to keep it seperate as I usually do tail -n 1 MYSTDOUTFILE to see if I was able to run to completion and then sort my STDERRFILES using ls -lS jobname.ejobid* by size to see if the stderr produced larger output than normal. this typically means there where errors on the run. This doesn't work though when the programmers are sloppy and lazy and output stdout to stderr. :(